<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="K10185" failures="2" tests="2" name="Test" time="46.996" errors="0" timestamp="2022-05-14T20:35:04 IST">
  <testcase classname="com.tmb.tests.CreateUserTest" name="createUserTest" time="25.219">
    <failure type="io.restassured.module.jsv.JsonSchemaValidationException" message="java.io.FileNotFoundException: src\test\resources\jsonschemas\user.json (The system cannot find the path specified)">
      <![CDATA[io.restassured.module.jsv.JsonSchemaValidationException: java.io.FileNotFoundException: src\test\resources\jsonschemas\user.json (The system cannot find the path specified)
at io.restassured.module.jsv.JsonSchemaValidator$JsonSchemaValidatorFactory.create(JsonSchemaValidator.java:286)
at io.restassured.module.jsv.JsonSchemaValidator.matchesJsonSchema(JsonSchemaValidator.java:157)
at com.tmb.assertions.ResponseAssert.hasExpectedResponseJsonSchema(ResponseAssert.java:47)
at com.tmb.tests.CreateUserTest.createUserTest(CreateUserTest.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.io.FileNotFoundException: src\test\resources\jsonschemas\user.json (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:211)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:153)
at com.github.fge.jackson.JsonLoader.fromFile(JsonLoader.java:165)
at io.restassured.module.jsv.JsonSchemaValidator$3.createSchemaInstance(JsonSchemaValidator.java:155)
at io.restassured.module.jsv.JsonSchemaValidator$3.createSchemaInstance(JsonSchemaValidator.java:152)
at io.restassured.module.jsv.JsonSchemaValidator$JsonSchemaValidatorFactory.create(JsonSchemaValidator.java:284)
... 31 more
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- createUserTest -->
  <testcase classname="com.tmb.tests.CreateUserTest" name="createUserTest" time="21.542">
    <failure type="io.restassured.module.jsv.JsonSchemaValidationException" message="java.io.FileNotFoundException: src\test\resources\jsonschemas\user.json (The system cannot find the path specified)">
      <![CDATA[io.restassured.module.jsv.JsonSchemaValidationException: java.io.FileNotFoundException: src\test\resources\jsonschemas\user.json (The system cannot find the path specified)
at io.restassured.module.jsv.JsonSchemaValidator$JsonSchemaValidatorFactory.create(JsonSchemaValidator.java:286)
at io.restassured.module.jsv.JsonSchemaValidator.matchesJsonSchema(JsonSchemaValidator.java:157)
at com.tmb.assertions.ResponseAssert.hasExpectedResponseJsonSchema(ResponseAssert.java:47)
at com.tmb.tests.CreateUserTest.createUserTest(CreateUserTest.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.io.FileNotFoundException: src\test\resources\jsonschemas\user.json (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:211)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:153)
at com.github.fge.jackson.JsonLoader.fromFile(JsonLoader.java:165)
at io.restassured.module.jsv.JsonSchemaValidator$3.createSchemaInstance(JsonSchemaValidator.java:155)
at io.restassured.module.jsv.JsonSchemaValidator$3.createSchemaInstance(JsonSchemaValidator.java:152)
at io.restassured.module.jsv.JsonSchemaValidator$JsonSchemaValidatorFactory.create(JsonSchemaValidator.java:284)
... 31 more
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- createUserTest -->
</testsuite> <!-- Test -->
